stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -218,121 +229,132 @@ function externCrc32b(byte[] input) returns handle = @java:Method {
 public function verifyRsaSha1Signature(byte[] data, byte[] signature, PublicKey publicKey)
                                        returns boolean|Error = @java:Method {
     name: "verifyRsaSha1Signature",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Verify RSA-SHA256 based signature.
+# ```ballerina boolean|crypto:Error validity = crypto:verifyRsaSha256Signature(data, signature, publicKey); ```
 #
 # + data - The content to be verified
 # + signature - Signature value
 # + publicKey - Public key used for verification
-# + return - Validity of the signature or `Error` if public key is invalid
+# + return - Validity of the signature or `crypto:Error` if public key is invalid
 public function verifyRsaSha256Signature(byte[] data, byte[] signature, PublicKey publicKey)
                                          returns boolean|Error = @java:Method {
     name: "verifyRsaSha256Signature",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Verify RSA-SHA384 based signature.
+# ```ballerina boolean|crypto:Error validity = crypto:verifyRsaSha384Signature(data, signature, publicKey); ```
 #
 # + data - The content to be verified
 # + signature - Signature value
 # + publicKey - Public key used for verification
-# + return - Validity of the signature or `Error` if public key is invalid
+# + return - Validity of the signature or `crypto:Error` if public key is invalid
 public function verifyRsaSha384Signature(byte[] data, byte[] signature, PublicKey publicKey)
                                          returns boolean|Error = @java:Method {
     name: "verifyRsaSha384Signature",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Verify RSA-SHA512 based signature.
+# ```ballerina boolean|crypto:Error validity = crypto:verifyRsaSha512Signature(data, signature, publicKey); ```
 #
 # + data - The content to be verified
 # + signature - Signature value
 # + publicKey - Public key used for verification
-# + return - Validity of the signature or `Error` if public key is invalid
+# + return - Validity of the signature or `crypto:Error` if public key is invalid
 public function verifyRsaSha512Signature(byte[] data, byte[] signature, PublicKey publicKey)
                                          returns boolean|Error = @java:Method {
     name: "verifyRsaSha512Signature",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Read a private key from the provided PKCS#12 archive file.
+# ```ballerina crypto:PrivateKey|crypto:Error privateKey = crypto:decodePrivateKey(keyStore, keyAlias, keyPassword); ```
 #
 # + keyStore - Key store or Trust store configurations
 # + keyAlias - Key alias
 # + keyPassword - Key password
-# + return - Reference to the private key or `Error` if private key was unreadable
+# + return - Reference to the private key or `crypto:Error` if private key was unreadable
```

Comment:
```suggestion
# + return - Reference to the private key or `crypto:Error` if private key was unreadable
```
Please refer the template for the correct format.

Reply:
Corrected.